index.js

```diff
@@ -6,8 +6,10 @@ var env = process.env.NODE_ENV || 'development';
 // makes it ok to check in your .env.development files
 // don't check in your .env.local file tho :)
 if (env === 'production') {
-    return;
+    loader.inject_env('.env.local');
```

Comment:
why not just move this above the `if` statement altogether and remove the else (leaving the return inside the if statement)